### ISSUE TITLE
Require a month and day for delayed release setting for at the collection level

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#961]
When a collection manager selects the release option of "Delay release until.." they should be required to enter a year, month, and day.

 ![Screen Shot 2021-01-28 at 2.19.07 PM.png](https://images.zenhubusercontent.com/5e9e06e81c23039800e23efc/8f78061e-16c5-46ad-8339-bc517313ff6b)

Currently, entering just a year is adequate and the deposit will complete successfully.

**Comment by peetucket** [comment]
This may not be needed if we implement the UI shown in #966

**Comment by jcoyne** [comment]
@amyehodge @peetucket wouldn't it still be needed for this option?

<img width="692" alt="Screen Shot 2021-02-08 at 1 04 51 PM" src="https://user-images.githubusercontent.com/92044/107268817-500c4580-6a0e-11eb-821b-7a73aea76f09.png">

**Comment by amyehodge** [comment]
@jcoyne @peetucket That option is a drop-down list with "6 months/1 year/2years/3 years" options. The collection manager is not selecting specific dates here.

**Comment by jcoyne** [comment]
@amyehodge so you would agree that we need to do either #966 or this one, but not both?

**Comment by jcoyne** [comment]
@amyehodge I think I might be confused. Is this ticket for the deposit form or on the collection form?

**Comment by amyehodge** [comment]
@jcoyne Collection form.

**Comment by jcoyne** [comment]
@amyehodge In that case I'm definitely confused because #966 doesn't have "month" or "day" fields.

**Comment by amyehodge** [comment]
Yes, that's correct. 

This ticket was filed because of the bug I was seeing in the current implementation, which has the fixed date option. However, since I filed this ticket, we had discussions about the mismatch with existing Hydrus collection settings and we opted to change this to the drop-down option of fixed range of time instead of fixed dates. So, we do not need to do this ticket if we get #966 implemented. Hence the comments above.
